02_configure_host.sh

```diff
@@ -40,6 +40,11 @@ if ! sudo iptables -C INPUT -i baremetal -p udp -m udp --dport 6230:6235 -j ACCE
     sudo iptables -I INPUT -i baremetal -p udp -m udp --dport 6230:6235 -j ACCEPT
 fi
 
+#Allow access to dualboot.ipxe
+if ! sudo iptables -C INPUT -p tcp --dport 80 -j ACCEPT ; then
+    sudo iptables -I INPUT -p tcp --dport 80 -j ACCEPT
```

Comment:
```suggestion
    sudo iptables -I INPUT -p tcp --dport 80 -m comment --comment "Serving pxe files" -j ACCEPT
```

Reply:
It's not only iPXE files, we also need it to download the image to the IPA ramdisk, so personally I'm fine with leaving out the comment, port 80 is ubiquitous enough most folks should figure out the kind of traffic it's used for :)